test-cases/tests/performance/k03-run-performance-test-against-rhsso.md

```diff
@@ -14,6 +14,7 @@ products:
     targets:
       - 0.1.0
       - 0.2.0
+      - 1.0.0
```

Comment:
same with this one?

Reply:
we have a separate test case actually. https://github.com/integr8ly/integreatly-operator/blob/master/test-cases/tests/performance/k04-run-performance-test-against-rhoam-3scale-user-sso.md